controllers/apps/v1beta3/emqx_handler.go

```diff
@@ -516,9 +523,9 @@ func generateInitPluginList(instance appsv1beta3.Emqx, existPluginList *appsv1be
 				PluginName: "emqx_management",
 				Selector:   instance.GetLabels(),
 				Config: map[string]string{
-					"management.listener.http":              "8081",
-					"management.default_application.id":     "admin",
-					"management.default_application.secret": "public",
+					"management.listener.http":              DefaultManagePort,
+					"management.default_application.id":     userName,
+					"management.default_application.secret": password,
```

Comment:
I think we don't need create management and dashboard plugin, because they are not update